src/sass/components/_post-box.scss

```diff
@@ -51,4 +63,26 @@
       padding-top: 30px;
     }
   }
+
+}
+
+.post-box {
+
+  &:hover {
+    text-decoration: none;
```

Comment:
It's default behavior - we don't need to override it again.

Reply:
Fixed in new commit